src/psbt.h

```diff
@@ -429,6 +429,11 @@ struct PSBTInput
                     std::vector<unsigned char> sig;
```

Comment:
Note for this commit: 14e35e675c576c770641160e982713dda4032bd8

I _think_ this is added because later in the commit 94fb539f4e22ed4ec9a65f90db26cb81e5b8749a, the last element of the partial sig is used to check for the sighash type match while signing the PSBT input for which this signature validity guarantee is needed. 

It'd be nice to mention something around this in the commit message to aid review (& for future reference).

Reply:
Added a sentence to the commit message.